docs/flashbots-mev-share/searchers/tutorials/flash-loan-arbitrage/flash-loan-basics.mdx

```diff
@@ -0,0 +1,31 @@
+import RemoteCodeBlock from "@site/src/components/RemoteCodeBlock"
+
+# Flash Loan Basics (Balancer)
+
+To recap, flash loans allow us to borrow vast amounts of tokens, which we can use to pay for trades. If we make a profit from executing an arbitrage, we pay the loan back and keep what's left over. If the arbitrage fails or does not produce enough profit to repay the loan, the transaction reverts and Flashbots prevents it from landing onchain, so we don't pay any fees. The only fees we pay are gas fees for successful arbitrage trades.
+
+We'll implement flash loans using Balancer, because they don't charge any loan fees, and have plenty of capital with which we can perform our arbitrage trades. Typically, flash loans (including Balancer) work by using callbacks. Callbacks are just functions that _other_ smart contracts call on _your_ smart contract. A callback in solidity is implemented by declaring a function in your smart contract with the exact name of the callback.
+
+In the case of a flash loan on Balancer, which we'll implement shortly, the callback we have to create is the `receiveFlashLoan` function. When we want to get a flash loan, we'll call `vault.flashLoan`, where `vault` is the Balancer Vault smart contract, which holds the capital used for flash loans. In turn, the vault contract will call the `receiveFlashLoan` callback on our smart contract; this is where we'll put the code that performs an arbitrage trade and pays back the loan. That's all there is to it!
```

Comment:
Nice! I think you explain how to use them really well here! It might be worth adding a paragraph on how and why they work if you feel inspired to talk briefly about transaction atomicity or call frames.

Reply:
added a lil paragraph, tried to keep it lingo-light